### PR TITLE
improved support for openemr package building in flex dockers and prepping in 5.0.2 docker

### PR DESCRIPTION
**Pull request by bradymiller**

### CODE REVIEW

File: docker/openemr/flex-3.7/Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN apk add --no-cache \
     php7-json php7-pdo php7-pdo_mysql php7-curl php7-ldap php7-openssl php7-iconv \
     php7-xml php7-xsl php7-gd php7-zip php7-soap php7-mbstring php7-zlib \
     php7-mysqli php7-sockets php7-xmlreader php7-redis php7-simplexml php7-xmlwriter \
-    perl mysql-client tar curl imagemagick-dev \
+    perl mysql-client tar curl imagemagick-dev nodejs \
```
Comment (d3sandoval) [inline docker/openemr/flex-3.7/Dockerfile]: Doesn't this alos need `nodejs-npm` ?

**Comment by bradymiller** [comment]
regarding nodejs-npm, this wasn't required in prior alpine (turns out it was aline 3.5, so will add this):
https://superuser.com/questions/1125969/how-to-install-npm-in-alpine-linux

**Comment by bradymiller** [comment]
Testing well and will bring in (note travis is intermittently breaking and not related to this code)

**Comment by bradymiller** [comment]
@d3sandoval ,
I just updated the flex dockers (5.0.2) with these updates :)

**Pull request merged**